interpreter/cling/lib/Interpreter/CIFactory.cpp

```diff
@@ -114,6 +114,23 @@ namespace {
                                        llvm::SmallVectorImpl<char>& Buf,
                                        AdditionalArgList& Args,
                                        bool Verbose) {
+    // For power-users: Let's see if the path is available as a predefined env
+    // variable to save repeated system calls when ROOT/CLING is initialized for each
+    // process in a many process system. CLING_CPPSYSINCL should contain paths
+    // separated by a ":" and otherwise contain the same paths as returned from
+    // the CppInclQuery further below.
```

Comment:
```suggestion
    // For power-users: Let's see if the path is available as a predefined env
    // variable to save repeated system calls when ROOT/CLING is initialized for
    // each process in a many process system. CLING_CPPSYSINCL should contain
    // paths separated by a ":" and otherwise contain the same paths as returned
    // from the CppInclQuery further below.
```